loki-logback-appender/src/test/java/com/github/loki4j/logback/ApacheHttpAppenderTest.java

```diff
@@ -52,4 +52,19 @@ public void testApacheHttpSend() {
         });
     }
 
+    @Test
+    public void testApacheHttpSendWithTenantHeader() {
```

Comment:
We can drop this test in favor of integration test

Reply:
I would like to keep it as unit test also. Just in case if someone can not run integration tests during development cycle